### DISCUSSION TITLE
[PSD1] Exercise 1.29

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#122] [Esercizi (A-L)]
**Exercise 1.29** Convert the following decimal numbers to 8-bit sign-magnitude number
(a) $42_{10}$
(b) $−63_{10}$
(c) $124_{10}$
(d) $−128_{10}$
(e) $133_{10}$

**Comment by Elia-Belli** [comment]
(a) $42_{10} = 00101010_2$
(b) $-63_{10} = 10111111_2$
(c) $124_{10} = 01111100_2$
(d) $-128_{10} = 110000000_2$ (Overflow: 8bit non sono sufficienti)
(e) $133_{10} = 010000101_2$ (Overflow: 8bit non sono sufficienti)

**Comment by niccolo-fato** [comment]
![photo_2024-11-15_17-52-26](https://github.com/user-attachments/assets/2956deb3-f523-4e88-a900-87dd0a782659)
